### PR TITLE
odin recipe project

### PR DESCRIPTION
**Pull request by abhishek310001**
Odin Recipes Project
# Description:
In this PR a page is designed such that it contain multiple links to their specific pages with the use of **anchor** tag.
I have done this with basic concepts of HTML in this project.

Live link: https://abhishek310001.github.io/odin-recipes/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ol>
+        <li><a href = "./recipes/pizza.html">Pizza</a></li>
+        <li><a href = "./recipes/butterchicken.html">Butter Chicken</a></li>
+        <li><a href="./recipes/kheer.html">Kheer</a></li>
+
```
Comment (devanuj97) [inline index.html]: remove this extra line

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ol>
+        <li><a href = "./recipes/pizza.html">Pizza</a></li>
+        <li><a href = "./recipes/butterchicken.html">Butter Chicken</a></li>
```
Comment (devanuj97) [inline index.html]: `href = "./recipes/butterchicken.html"` => `href="./recipes/butterchicken.html"` 

Remove extra space, same for line no 12

File: recipes/kheer.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <li>½ cup basmati rice</li>
+        <li>¼ cup raisins</li>
+        <li>½ teaspoon ground cardamom</li>
+        <li>½ teaspoon rose water (Optional)</li>
+        <li>¼ cup sliced almonds, toasted</li>
+        <li>¼ cup chopped pistachio nuts</li>
+        
+    </ul>
+    <h2>Steps</h2>
+    <ol>
```
Comment (devanuj97) [inline recipes/kheer.html:29]: indentation issue, please fix this

File: recipes/kheer.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (devanuj97) [inline recipes/kheer.html]: provide title of the page relevant to the page context

File: index.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (devanuj97) [inline index.html]: provide title of the page relevant to the page context

File: recipes/butterchicken.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Butter Chicken Recipe</h1>
+    <img src = "../images/butterchicken.webp">
```
Comment (devanuj97) [inline recipes/butterchicken.html]: remove extra space 
`` => ``

File: recipes/kheer.html [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+</head>
+<body>
+    <h1>Kheer Recipe</h1>
+    <img src = "../images/kheer.webp">
```
Comment (devanuj97) [inline recipes/kheer.html]: remove extra space from here as well, when you are assigning image url in src

File: recipes/kheer.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+        <li>2 cups coconut milk</li>
+        <li>2 cups milk</li>
+        <li>3 tablespoons white sugar</li>
+        <li>½ cup basmati rice</li>
+        <li>¼ cup raisins</li>
+        <li>½ teaspoon ground cardamom</li>
+        <li>½ teaspoon rose water (Optional)</li>
+        <li>¼ cup sliced almonds, toasted</li>
+        <li>¼ cup chopped pistachio nuts</li>
+
```
Comment (devanuj97) [inline recipes/kheer.html]: remove extra line

File: recipes/pizza.html [hunk 9]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (devanuj97) [inline recipes/pizza.html]: Provide title name relevant to page context

File: recipes/pizza.html [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+</head>
+<body>
+    <h1>Pizza Recipe</h1>
+    <img src = "../images/pizza.jpeg">
```
Comment (devanuj97) [inline recipes/pizza.html]: Remove extra space from here as well

File: recipes/pizza.html [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+        <li>2 teaspoons extra-virgin olive oil, divided</li>
+        <li>1 clove garlic</li>
+        <li>1 pinch sea salt</li>
+        <li>6 ounces fresh mozzarella cheese, chopped</li>
+        <li>1/4 cup shredded Parmigiano-Reggiano cheese</li>
+        <li>10 leaf basil leaves, torn</li>
+    </ul>
+    <h2>Steps</h2>
+    <ol>
+        <li>Preheat the oven to 425 degrees F (220 degrees C). Spread each sheet of puff pastry onto a piece of lightly floured parchment paper. Use a rolling pin to roll pastry into even rectangles. Lift pastry (with parchment paper) onto baking sheets.
```
Comment (devanuj97) [inline recipes/pizza.html]: Change to this for other place as well where `<li>` tag can not fit in one line
```
<li>
Preheat the oven to 425 degrees F (220 degrees C). Spread each sheet of puff pastry onto a piece of lightly floured parchment paper. Use a rolling pin to roll pastry into even rectangles. Lift pastry (with parchment paper) onto baking sheets.
</li>
```

File: recipes/pizza.html [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+        </li>
+        <li>Pierce puff pastry all over with a fork. Beat egg with water in a small dish; brush pastry all over with egg wash.</li>
+        <li>Bake in the preheated oven for 10 minutes.</li>
+        <li>Meanwhile, pulse tomatoes, 1/2 teaspoon of the olive oil, garlic, and a pinch of salt together in a food processor until pureed.</li>
+        <li>Carefully press pastry crusts down to deflate them. Spread half the tomato sauce evenly to within about an inch of edges of each crust; drizzle with remaining olive oil. Top evenly with mozzarella cheese and Parmigiano-Reggiano cheese.</li>
```
Comment (devanuj97) [inline recipes/pizza.html]: ` Parmigiano-Reggiano` please, use plan english this is more readable

File: recipes/butterchicken.html [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ButterChicken</title>
+</head>
+<body>
+    <h1>Butter Chicken Recipe</h1>
+    <img src="../images/butterchicken.webp">
```
Comment (balwant-jtc) [inline recipes/butterchicken.html]: An `img` element must have an 'alt' attribute.

**Review by balwant-jtc — CHANGES_REQUESTED** [review]

File: recipes/butterchicken.html (hunk 13 above)
Comment (devanuj97) [reply recipes/butterchicken.html]: +1

File: index.html [hunk 14]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>OdinRecipesProject</title>
```
Comment (devanuj97) [inline index.html]: Make this space separated, which is more human readable => `Odin Recipes Dashboard`

File: recipes/butterchicken.html [hunk 15]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ButterChicken</title>
```
Comment (devanuj97) [inline recipes/butterchicken.html]: `ButterChicken`  => `Butter Chicken`

**Review by devanuj97 — APPROVED** [review]

**Comment by devanuj97** [comment]
@abhishek310001, please change file name `recipes/butterchicken.html` => `recipes/butter-chicken.html`

html file name should be hyphen separated

**Comment by abhishek310001** [comment]
will fix these issues ASAP

**Pull request merged**